include/odp/api/spec/pool.h

```diff
@@ -214,6 +222,15 @@ typedef struct odp_pool_param_t {
 			    defined by pool capability pkt.max_uarea_size.
 			    Specify as 0 if no user area is needed. */
 			uint32_t uarea_size;
+
+			/** Minimum Headroom size in bytes. Minimum number of
+			    bytes of space preceding packet data to reserve for
+			    use as headroom. Each newly allocated packet from
+			    the pool must have atleast this much headroom.
+			    Must not exceed the implementation defined
+			    'max_headroom' in odp_pool_capability().
+			    Implementation may round up to a higher value. */
+			uint32_t min_headroom;
```

Comment:
I'd still like to see a usage note somewhere indicating that setting `min_headroom` to 0 means that the application does not require any packet headroom so the implementation may optimize pool storage usage accordingly.

Reply:
Okay